pkg/features/kube_features.go

```diff
@@ -1045,8 +1045,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	genericfeatures.ServerSideApply: {Default: true, PreRelease: featuregate.GA},
 
-	genericfeatures.ServerSideFieldValidation: {Default: false, PreRelease: featuregate.Alpha},
-
+	genericfeatures.ServerSideFieldValidation: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
nit: keep the newline after this (keeping them separate keeps indentation independent)